JSON/Hereditary Base Notation.json

```diff
@@ -14,5 +14,6 @@
   "Souvenir": { "Status": "Supported" },
   "SteamID": "1942342709",
   "Symbol": "Hbn",
+  "TutorialVideoUrl": "https://www.youtube.com/watch?v=cZKndtREXn4&t=2s",
```

Comment:
This link will make the tutorial video starts at 2 seconds.

Reply:
First two seconds is just an opening sound; I'd argue it's a good use of timeskip.

Reply:
I think it is better for it to be consistent across modules.